README.rst

```diff
@@ -86,13 +86,19 @@ Using environment files (including any SSL settings) in ``~/.irods/``:
 ... except KeyError:
 ...     env_file = os.path.expanduser('~/.irods/irods_environment.json')
 ...
->>> ssl_context = ssl.create_default_context(purpose=ssl.Purpose.SERVER_AUTH, cafile=None, capath=None, cadata=None)
->>> ssl_settings = {'ssl_context': ssl_context}
->>> with iRODSSession(irods_env_file=env_file, **ssl_settings) as session:
+>>> ssl_settings = {} # Or, optionally: {'ssl_context': <user_customized_SSLContext>}
+>>> with iRODSSession(irods_env_file=env_file, ** ssl_settings) as session:
```

Comment:
I noticed the new space between `**` and the parameter name... is this more standard/pythonic?

Both here and in other places in this PR...

Reply:
No, not standard or pythonic at all. But there seemed no other way to prevent either my editor from having the syntax highlighting screw up in vim, or the markup on Github doing similar weird stuff.   I thought the new space  looked pretty neutral or maybe even slightly positive effect on source readability, but then I tend to like more whitespace than others do.

I don't mind changing this back , if it's what we prefer.

Reply:
consider running things through python black... see what it does...

Reply:
black definitely prefers **d, not ** d

Reply:
yeah, okay... let's go with no space for now, as it's a change from what we've done before... and not what the most popular formatter does.

Reply:
reverted them